best-practices/javascript_debugging.md

```diff
@@ -0,0 +1,8 @@
+# Debugging in Javascript
+
+If you'd like to use stepwise debugging in javascript, consider using the following procedure when working in VSCode.
+
+* Type <CMD><Shift>p to bring up the command palette
```

Comment:
```suggestion
* Type `<CMD><Shift>p` to bring up the command palette
```

looks like markdown tries to render the parts in angle brackets as tags (if you "view file" on this).  hopefully this suggested edit takes care of that?